spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'the signin process', type: :feature do
+  describe 'login page' do
+    before do
+      visit home_path
+      User.create(name: 'TestUser', email: 'test@user.com', password: 'testu1', password_confirmation: 'testu1',
+                  confirmed_at: Date.today)
+    end
+
+    it 'shows the username and password inputs' do
+      expect(page).to have_css('input[type="email"]')
+      expect(page).to have_css('input[type="password"]')
+    end
+
+    it 'shows the "Submit" button' do
+      expect(page).to have_button('Log in')
+    end
+
+    it 'click the submit button without filling in the email and the password shows the Error message' do
+      click_button 'Log in'
+      expect(page).to have_content('Invalid Email or password')
+    end
+
+    it 'filling email and password fields with incorrect data shows the Error message' do
+      fill_in 'Email', with: 'user@example.com'
+      fill_in 'Password', with: 'password'
+      click_button 'Log in'
+      expect(page).to have_content('Invalid Email or password')
+    end
+
+    it 'filling email and password fields with correct data redirects to the root page' do
+      fill_in 'Email', with: 'test@user.com'
+      fill_in 'Password', with: 'testu1'
+      click_button 'Log in'
+      expect(page).to have_current_path(home_path)
+    end
+  end
+end
+
+RSpec.describe 'list of users in', type: :feature do
```

Comment:
* Kindly add the `Tests that asserts you can see the profile picture for each user ` as it is required in the project requirements.